### PR TITLE
Update docs of Visual Studio

### PR DESCRIPTION
**Pull request by AnirudhPanda**
Changes done to docs of Visual Studio!
@HarshCasper @shubhamkarande13

### CODE REVIEW

File: docs/prerequisites/visual_studio.rst [hunk 1]
```diff
@@ -20,6 +20,35 @@ For Visual Studio 2019
...
 
+After launching the Visual Studio installer:
+
+-  Select the desktop development with C++ workload and make sure it is
+   checked.
+-  Make sure the **Install while downloading** option is selected for
+   faster installation.
+-  Click on **Install**
+
+.. figure:: ../images/visual_studio/install_page.PNG
```
Comment (HarshCasper) [inline docs/prerequisites/visual_studio.rst:31]: This picture is quite blurred and barely anything can be captured from it.

File: docs/prerequisites/visual_studio.rst [hunk 2]
```diff
@@ -20,6 +20,35 @@ For Visual Studio 2019
...
+
+After the installation is complete:
+
+-  Restart your system.
+-  Create a folder and name it **Development**.
+-  Inside the **Development** folder create a folder and name it
+   **moja-global**
+-  Open the moja-global folder in the command prompt.
+
+.. figure:: ../images/visual_studio/command_prompt.PNG
```
Comment (HarshCasper) [inline docs/prerequisites/visual_studio.rst:45]: This picture is quite blurred and barely anything can be captured from it.

File: docs/prerequisites/visual_studio.rst [hunk 3]
```diff
@@ -20,6 +20,35 @@ For Visual Studio 2019
...
+.. figure:: ../images/visual_studio/install_page.PNG
+   :alt: Install Section of Visual Studio 
+   :align: center
+
+   Install Section of Visual Studio 2019
+
+After the installation is complete:
+
+-  Restart your system.
+-  Create a folder and name it **Development**.
```
Comment (HarshCasper) [inline docs/prerequisites/visual_studio.rst:40]: Is there any preferred drive where this folder should be created? If yes, please add that.

File: docs/prerequisites/visual_studio.rst [hunk 4]
```diff
@@ -20,6 +20,35 @@ For Visual Studio 2019
...
+-  Inside the **Development** folder create a folder and name it
```
Comment (HarshCasper) [inline docs/prerequisites/visual_studio.rst:41]: Keep `Development` and `moja-global` in backticks, not bold

File: docs/prerequisites/visual_studio.rst [hunk 5]
```diff
@@ -20,6 +20,35 @@ For Visual Studio 2019
...
+   **moja-global**
+-  Open the moja-global folder in the command prompt.
```
Comment (HarshCasper) [inline docs/prerequisites/visual_studio.rst:43]: Keep `moja-global` in backticks

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: docs/prerequisites/visual_studio.rst (hunk 3 above)
Comment (shubhamkarande13) [reply docs/prerequisites/visual_studio.rst]: C:/ should be ideal as it is default for most configurations and used in the video.

Comment (chicken-biryani) [reply docs/prerequisites/visual_studio.rst]: I don't think it makes a difference.

**Review by chicken-biryani — APPROVED** [review]

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**